### PR TITLE
chore(amazon): add zen 3 to us, es, and nl

### LINKED ISSUE #740: Ryzen 5600X is not being checked on Amazon.com
### Description

I have `amazon` listed as one of the stores but it's not being checked for 5600X

### PR DESCRIPTION
**Pull request by picchioni**
### Description

Fixes #740 in addition to adding missing Ryzen CPUs to other regional Amazon sites

### Testing

1. Navigate to various Amazon product pages to confirm product exists and add to card functionality works

### New dependencies

None

### CODE REVIEW

File: src/store/model/amazon-es.ts [hunk 1]
```diff
@@ -99,6 +99,34 @@ export const AmazonEs: Store = {
 			model: 'trinity',
 			series: '3080',
 			url: 'https://www.amazon.es/dp/B08HR1NPPQ'
+		},
+		{
+			brand: 'amd',
+			cartUrl: 'https://www.amazon.es/gp/aws/cart/add.html?ASIN.1=B08166SLDF&Quantity.1=1',
+			model: '5600x',
+			series: 'ryzen5600',
+			url: 'https://www.amazon.es/dp/product/B08166SLDF'
```
Comment (jef) [inline src/store/model/amazon-es.ts:108]:
```suggestion
			url: 'https://www.amazon.es/dp/B08166SLDF'
```

Do you mind cleaning this up for the rest of the URLs?

**Review by jef — CHANGES_REQUESTED** [review]
Just a small change, otherwise looks great! I appreciate the contribution!

File: src/store/model/amazon-es.ts (hunk 1 above)
Comment (picchioni) [reply src/store/model/amazon-es.ts:108]: @jef Sure thing, I'll get that cleaned up later this evening.

Comment (jef) [reply src/store/model/amazon-es.ts:108]: Thanks!

**Comment by picchioni** [comment]
Hey @jef I cleaned up the URLs, and added cartURL for outstanding products across all Amazon regions. I noticed after committing, the files use tabs instead of spaces for indentation which threw off the formatting. I was going to convert the tabs to 4 spaces per tab but wanted to check with you first, otherwise I'll convert my code to tabs. Thoughts?

**Comment by jef** [comment]
The linter will fix this for you. Just run `npm run lint:fix`

**Review by jef — APPROVED** [review]

**Pull request merged**